spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -268,15 +273,20 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			// and expand them to their long form if applicable
 			AppDefinition revisedDefinition = mergeAndExpandAppProperties(currentApp, metadataResource, appDeployTimeProperties);
 			AppDeploymentRequest request = new AppDeploymentRequest(revisedDefinition, appResource, deployerDeploymentProperties);
-
 			try {
+				String loggingString = String.format("Deploying the app '%s.%s'", currentApp.getStreamName(),
```

Comment:
I would change the format to not be <streamName>.<appLabel> and be explicit in terms of the components.  

```
Deploying application log as part of stream ticktock using the resource maven://
```

I was thinking if we could get finer grained in terms of the download request, that would live in the deployer and the call to `getInputStream`....